real_time/surface_src/java_src/Alice/src/test/java/com/roboclub/robobuggy/nodes/localizers/MultipleIMUtest.java

```diff
@@ -0,0 +1,395 @@
+package com.roboclub.robobuggy.nodes.localizers;
+
+import com.hcrest.jfreespace.Device;
+import com.hcrest.jfreespace.FreespaceErrorCodes;
+import com.hcrest.jfreespace.inreport.DecoderFactory;
+import com.hcrest.jfreespace.inreport.FreespaceMsgInMotionEngineOutput;
+import com.hcrest.jfreespace.inreport.HidInMsg;
+import com.hcrest.jfreespace.outreport.HidOutMsg;
+import com.roboclub.robobuggy.messages.*;
+import com.roboclub.robobuggy.nodes.sensors.HillCrestImuNode;
+import com.roboclub.robobuggy.ros.*;
+import org.junit.Test;
+
+import java.util.ArrayList;
+import java.util.concurrent.TimeUnit;
+
+/**
+ * Created by Alex on 11/10/2016.
+ */
+public class MultipleIMUtest {
+    private Subscriber accSub = new Subscriber("test", NodeChannel.IMU_ACCELERATION.getMsgPath(), (topicName, m) -> {
```

Comment:
please try to initialize this field in `oneTimeSetup()` instead of at the top. It makes the code much easier to read